unattended_scripts/install_functions/opendistro/kibana.sh

```diff
@@ -96,6 +99,7 @@ copyKibanacerts() {
     if [ -d "${base_path}/certs" ]; then
         eval "cp ${base_path}/certs/kibana* /etc/kibana/certs/ ${debug}"
         eval "cp ${base_path}/certs/root-ca.pem /etc/kibana/certs/ ${debug}"
+        
```

Comment:
This line is empty, remove it

Reply:
Done in https://github.com/wazuh/wazuh-packages/commit/39c0d9730785973b0736a4b86c2994b9917ac73d